.env

```diff
@@ -13,13 +13,19 @@ NGINX_PORT_HOST=81
 NGINX_SSLPORT_HOST=8443
 
 # CKAN databases
-POSTGRES_USER=ckan
-POSTGRES_PASSWORD=ckan
+POSTGRES_USER=postgres
+POSTGRES_PASSWORD=admindbpassword
+POSTGRES_DB=admindb
+PGDATA=/var/lib/postgresql/data/db
```

Comment:
I wouldn't change or set this unless you have a reason to, and the only reason that I've run across is to make the pgdata keyed to the postgresql version to make it much easier to do upgrades.

e.g., 
```
PGDATA=/var/lib/postgresql/VERSION/data
```

Reply:
Is that directory already setup as a database? i.e., does the initdb run before you are setting this?  Possibly as a cached volume mount?

Reply:
Looks like it was because there was a `pg_hba.conf` file located in that directory which caused problems with the mount. This is created during building the local db image (we were overriding this file). Looks like we are OK with the default `pg_hba.conf` file in the  base `postgres:12-alpine` image